tools/flang1/flang1exe/accpp.c

```diff
@@ -701,6 +701,20 @@ print_and_check(FILE *ff, char *str, char end_c)
     fputc(end_c, ff);
 }
 
+static void acc_display_macros()
+{
+  unsigned int i;
+  for (i = 1U; i < next_hash; ++i) {
+    char *name = &deftab[hashrec[i].name];
+    char *value = &deftab[hashrec[i].value];
+    if ((strcmp(name, "__LINE__") == 0) ||
+        (strcmp(name, "__DATE__") == 0) ||
+        (strcmp(name, "__FILE__") == 0) ||
+       ( strcmp(name, "__TIME__") == 0)) continue;
+    printf("%s : %s\n", name, value);
```

Comment:
Nit: I have not checked the code elsewhere, but do we use `printf` to usually display values? I would have assumed it will be `fprintf` or something like that.

Reply:
changed to `fprintf(gbl.outfil.....`
-->fixed